types.js

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
I think this is worth documenting somewhere.
```
/** @typedef {import('../types.js').SETTINGS} SETTINGS */

/** @type {SETTINGS} globalThis.vueSnapshots */
globalThis.vueSnapshots = {};
```

Though not sure if it would need

```
/** @typedef {import('../node_modules/vue3-snapshot-serializer/types.js').SETTINGS} SETTINGS */

/** @type {SETTINGS} globalThis.vueSnapshots */
globalThis.vueSnapshots = {};
```
or
```
/** @typedef {import('vue3-snapshot-serializer/types.js').SETTINGS} SETTINGS */

/** @type {SETTINGS} globalThis.vueSnapshots */
globalThis.vueSnapshots = {};
```
or just
```
/** @typedef {import('vue3-snapshot-serializer/types').SETTINGS} SETTINGS */

/** @type {SETTINGS} globalThis.vueSnapshots */
globalThis.vueSnapshots = {};
```

Guess I can bump and publish and manually test.